packages/react/src/auto/mui/MUIAutoForm.tsx

```diff
@@ -35,8 +35,12 @@ export const MUIAutoForm = <
 ) => {
   const { action, findBy } = props as MUIAutoFormProps<GivenOptions, SchemaT, ActionFunc> & { findBy: any };
 
+  if (!action) {
+    throw new Error("A valid Gadget action is required for AutoForm components");
+  }
```

Comment:
Could we just reuse the same `ApiTriggerRequiredError` message here?

Reply:
Updated to use the reference the same error message string